paymentsheet/src/androidTest/java/com/stripe/android/paymentelement/EmbeddedPaymentElementTestRunner.kt

```diff
@@ -73,6 +75,7 @@ internal fun runEmbeddedPaymentElementTest(
                 countDownLatch.countDown()
             },
         ).apply {
+            rowSelectionBehavior(rowSelectionBehavior)
```

Comment:
This shouldn't be needed. Callers of this function should be able to call the builder param above.